api/models/linux/namespace/namespace.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package linux.namespace;
+
+option go_package = "github.com/ligato/vpp-agent/api/models/linux/namespace;linux_namespace";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+option (gogoproto.messagename_all) = true;
+
+message NetNamespace {
+    enum ReferenceType {
+        UNDEFINED = 0;
+        NSID = 1;         /* named namespace */
+        PID = 2;          /* namespace of a given process */
+        FD = 3;           /* namespace referenced by a file handle */
```

Comment:
Is this really an FD?  Or is it a filename?

Reply:
^^^^^^

Reply:
This is actually file path, so perhaps `FILENAME` would be better?